darshan-util/pydarshan/devel/build-libdarshanutil.sh

```diff
@@ -2,7 +2,12 @@
 
 PREFIX=$PWD/devenv/libdarshanutil
 
-cd ../
+# go to repository root, and generate/update configure files
+cd ../../
+./prepare.sh
+
+# build darshan-utils
+cd darshan-util
```

Comment:
I stumbled upon this missing for heatmap support because of the need to update the configuration etc., but strictly speaking we could also break it out.

Reply:
If it's relevant to the PR then this can be resolved, it just wasn't obvious to me